lib/vernier/autorun.rb

```diff
@@ -13,6 +15,11 @@ class << self
     @options.transform_keys! do |key|
       key.sub(/\AVERNIER_/, "").downcase.to_sym
     end
+    if @options[:metadata]
+      @options[:user_metadata] = JSON.parse(Base64.decode64(@options[:metadata])).each_with_object({}) do |hash, result|
+        result[hash["key"].to_sym] = hash["value"]
+      end
```

Comment:
Nit.

```suggestion
      @options[:user_metadata] = JSON.parse(Base64.decode64(@options[:metadata])).to_h do |pair_hash|
        [pair_hash["key"].to_sym, pair_hash["value"]]
      end
```

It also seems like

Reply:
Applied in 4046798, but modified to not use "pair_hash", as we are no longer storing "key" and "value" separately.